### DISCUSSION TITLE
Window management

### DISCUSSION THREAD

**Discussion by Mikkel3** [plugdata-team/plugdata#732] [Ideas]
I noticed that Plugdata doesn't react to draging it to the side of the screen. Normally windows 10/11 will give the option to make it cover half the screen, but it doens't react to Plugdata. 
Furthermore when doing this with another program, windows usually gives the option to choose a program to split on the other half of screen, but it doesn't give the option to choose Plugdata. 
Screenshot shows how other programs react to this feature, Plugdata is opened but not available to use in this manner.

Would be cool if it could be made available! 

![Screenshot 2023-03-21 000002](https://user-images.githubusercontent.com/95369461/226484395-2afa745d-1300-4ca5-baef-ebc39f8425c9.png)

**Comment by timothyschoen** [comment]
Thanks for letting me know! There is a similar problem on Linux right now, and macOS too if install a tiling wm. If you use the OS titlebar (can be enabled in advanced settings), it will probably work. It would be great to fix this this for the custom titlebar too, but that would be a bit of work since it would need to be fixed for each OS individually.
